lib/environments/meteorEnvironment.js

```diff
@@ -308,6 +308,31 @@ export default {
     return moduleName;
   },
 
+  moduleSideEffectImports: ({ pathToCurrentFile, config }: Object): Array<string> => {
+    let basePath = '';
+    if (pathToCurrentFile.endsWith('.js')) {
+      basePath = pathToCurrentFile.slice(0, -3);
+    } else if (pathToCurrentFile.endsWith('.jsx')) {
+      basePath = pathToCurrentFile.slice(0, -4);
```

Comment:
Although since this is for the meteor environment, perhaps this doesn't apply? I don't have much context there, so I'll trust your judgement.